.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 | Q             | A
 | ------------- | ---
-| Branch?       | "master" for new features / 2.7, 2.8 or 3.1 for fixes
```

Comment:
Problem is that "master" is a common word in English. Removing the quotes may confuse the less experienced developers (although I'm divided myself about this).